apps/taxes/package.json

```diff
@@ -62,9 +59,9 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/react-hooks": "^8.0.1",
     "@types/node": "^18.8.1",
-    "@types/react": "^18.0.21",
+    "@types/react": "~18.0.38",
     "@types/react-dom": "^18.0.6",
-    "@vitejs/plugin-react": "^3.1.0",
+    "@vitejs/plugin-react": "^4.0.0",
```

Comment:
is this change required in this PR? We should update dependencies in all apps at the same time if possible